manuscript/11-SoftEngProfPr.txt

```diff
@@ -233,10 +230,10 @@ Machinery (ACM) και από το διοικητικό συμβούλιο τη
 Για αυτό τον λόγο συμμετέχουν και σε παρεμφερείς δραστηριότητες, όπως
 
 
-  * Την καθιέρωση και την διάδοση ενός σώματος γενικά αποδεκτής γνώσης
-  * Την διαπίστευση, την πιστοποίηση και την χορήγηση αδειών
-  * Την διακήρυξη πειθαρχικών μέτρων
-  * Την εξέλιξη του επαγγέλματος μέσω συνεδρίων, εκπαίδευσης και δημοσιεύσεων
+  * Την καθιέρωση και την διάδοση ενός σώματος γενικά αποδεκτής γνώσης,
+  * Την διαπίστευση, την πιστοποίηση και την χορήγηση αδειών,
+  * Την διακήρυξη πειθαρχικών μέτρων,
+  * Την εξέλιξη του επαγγέλματος μέσω συνεδρίων, εκπαίδευσης και δημοσιεύσεων.
```

Comment:
I agree with the comma.  But each line should then start with a lowercase letter.